packages/11ty/_plugins/figures/annotation-factory.js

```diff
@@ -0,0 +1,58 @@
+const chalkFactory = require('~lib/chalk')
+const path = require('path')
+const titleCase = require('~plugins/filters/titleCase')
+const { error } = chalkFactory('Figure Processing:IIIF:Annotations')
+
+/**
+ * Quire Annotation
+ */
+module.exports = class AnnotationFactory {
+  constructor(iiifConfig) {
+    this.iiifConfig = iiifConfig
+  }
+
+  create(figure, data) {
```

Comment:
JSDoc with a `typedef` for `figure` and possible the shape of `data` may be helpful.